gum/arch-arm/gumthumbrelocator.h

```diff
@@ -16,6 +16,14 @@ G_BEGIN_DECLS
 typedef struct _GumThumbRelocator GumThumbRelocator;
 typedef struct _GumThumbLocation GumThumbLocation;
 typedef struct _GumThumbITBlock GumThumbITBlock;
+typedef guint GumThumbRelocatorBranchType;
+
+enum _GumThumbRelocatorBranchType
+{
+  GUM_THUMBRELOCATOR_BRANCH_INVALID,
+  GUM_THUMBRELOCATOR_BRANCH_SHORT,
+  GUM_THUMBRELOCATOR_BRANCH_LONG
```

Comment:
How about `GumITBranchType`:

```suggestion
  GUM_IT_BRANCH_INVALID,
  GUM_IT_BRANCH_SHORT,
  GUM_IT_BRANCH_LONG
```